plugin.py

```diff
@@ -65,6 +73,20 @@ def handle_end_progress(self, params) -> None:
     def handle_report_progress(self, params: List[str]) -> None:
         self._start_indicator("{}: {}".format(self.package_name, "; ".join(params)))
 
+    @classmethod
+    def get_dev_environment(cls, settings: Optional[sublime.Settings] = None) -> str:
+        if not settings:
+            settings = sublime.load_settings(cls.package_name + ".sublime-settings")
+
+        return str(settings.get("dev_environment"))
+
+    @classmethod
+    def inject_extra_paths_st(cls, server_settings: Dict[str, Any]) -> None:
+        # add package dependencies into "python.analysis.extraPaths"
+        extraPaths = server_settings.get("python.analysis.extraPaths", [])  # type: List[str]
+        extraPaths.extend(cls.find_package_dependency_dirs())
+        server_settings["python.analysis.extraPaths"] = extraPaths
```

Comment:
Won't work properly because if `server_settings` comes from a `DottedDcit`, it would be in the form of 

```py
DottedDict(
    {
        "python": {
            "analysis": {
                "extraPaths": [
                    # ...
                ],
            },
        },
    }
)
```